src/main/java/pro/mikey/autoclicker/AutoClicker.java

```diff
@@ -216,10 +216,10 @@ private void keyInputEvent(MinecraftClient mc) {
         while (toggleHolding.wasPressed()) {
             this.isActive = !this.isActive;
             mc.player.sendMessage(
-                    (this.isActive ? Language.MSG_HOLDING_KEYS : Language.MSG_RELEASED_KEYS)
-                            .getText()
-                            .formatted(this.isActive ? Formatting.GREEN : Formatting.RED),
-                    true);
+                    (this.isActive ? Language.MSG_HOLDING_KEYS : Language.MSG_RELEASED_KEYS).getText(), true
```

Comment:
Does this mean you're not using the colors anymore?

Reply:
It does, I can go back and fix it but didn't see much of an issue with it being gone.

Reply:
Just user feedback, I'd prefer it is there, the MutableComponent (not sure what it is in yarn) should be pretty simple to add it back to.

Reply:
Alright fixed it, was quite simple like you said!